Tofu/AccountsViewController.swift

```diff
@@ -123,6 +135,26 @@ class AccountsViewController: UITableViewController {
             navigationItem.leftBarButtonItem = editButtonItem
         }
     }
+    
+    // MARK: UIImagePickerControlDelegate
+    
+    func imagePickerController(_ picker: UIImagePickerController,
+                               didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        guard let selectedQRCode = info[UIImagePickerController.InfoKey.originalImage] as? UIImage,
```

Comment:
If this guard fails, should we present a message to the user? Something like "Failed to Detect QR Code"?